### PR TITLE
docs: expose hasura on https (close #2956)

### LINKED ISSUE #2956: docs: update google cloud guide to expose hasura on https instead of http
The following document is a great guide to set up a k8s endpoint with load balancing, but the load balancer exposes only the port:80.

I'm having lot's of issues now trying to add https/ssl to the k8s cluster. I think it should be good to add documentation on the guide about it so it can be more complete. No one exposes a database-consuming service trough http these days.

https://github.com/hasura/graphql-engine/blob/master/docs/graphql/manual/guides/deployment/google-kubernetes-engine-cloud-sql.rst

### PR DESCRIPTION
**Pull request by vitaliytv**
### Affected components

- [ ] Docs

### CODE REVIEW

**Comment by marionschleifer** [comment]
@arvi3411301 can you please review this regarding technical correctness?

File: docs/graphql/manual/guides/deployment/google-kubernetes-engine-cloud-sql.rst [hunk 1]
```diff
@@ -185,6 +181,67 @@ command below. It usually takes a couple of minutes.
 Once the IP is allocated, visit the IP in a browser and it should open the
 console.
 
+Expose GraphQL engine (HTTPS)
+---------------------
+
+Let's expose Hasura with `Ingres
```
Comment (arvi3411301) [inline docs/graphql/manual/guides/deployment/google-kubernetes-engine-cloud-sql.rst:187]:
```suggestion
Let's expose Hasura with `Ingress 
```

File: docs/graphql/manual/guides/deployment/google-kubernetes-engine-cloud-sql.rst [hunk 2]
```diff
@@ -185,6 +181,67 @@ command below. It usually takes a couple of minutes.
...
+.. code-block:: yaml
+
+
+	apiVersion: v1
+	kind: Service
+	metadata:
+	  labels:
+	    app: hasura
+	  name: hasura
+	  namespace: default
```
Comment (arvi3411301) [inline docs/graphql/manual/guides/deployment/google-kubernetes-engine-cloud-sql.rst:199]:
```suggestion
```

**Comment by vitaliytv** [comment]
> @vitaliytv thank you for your contribution 🙏 I have two things to add to the review:
> 
> * Could you make the underlines of the headings the same length as the title itself?
> * The deployment guides have moved to a different section: https://hasura.io/docs/1.0/graphql/manual/deployment/deployment-guides/google-kubernetes-engine-cloud-sql.html. Would you mind adding it there?
>   Once this is done, we can get this merged 🙂

https://github.com/hasura/graphql-engine/pull/5770

correct ?

**Comment by marionschleifer** [comment]
@vitaliytv looks perfect, thank you so much 🙏 closing this, and taking the other one forward.

**Pull request closed without merging**